packages/consent-manager/index.js

```diff
@@ -27,7 +26,7 @@ export default class ConsentManager extends Component {
 
   saveAndLoadAnalytics(preferences) {
     if (typeof preferences === 'undefined') {
-      preferences = { All: false, 'Segment.io': false }
+      preferences = { loadAll: false, segment: false }
```

Comment:
You'll see this change here and in the tests - somehow the property names had drifted and were incorrect. The [`loadAnalytics` function](https://github.com/hashicorp/react-components/blob/8dfad01e23f24496f73952d3b7fc6b7c5f3c725c/packages/consent-manager/partials/load.js#L20-L33) (in `partials/load`) is looking for `loadAll`, `segment` and `custom` so have adjusted these appropriately here.